### DISCUSSION TITLE
Cannot clean up blobs data

### DISCUSSION THREAD

**Discussion by RickJou** [containerd/nerdctl#2050] [Q&A]
### Description

This directory takes up so much space, but I can't reclaim the space.

### Steps to reproduce the issue

[root@worker1-k8s sha256]# pwd
/var/lib/containerd/io.containerd.content.v1.content/blobs/sha256
[root@worker1-k8s sha256]# du -sh
132G	.

---
[root@worker1-k8s sha256]# crictl images
<img width="621" alt="image" src="https://user-images.githubusercontent.com/22700839/221169624-652f67e3-bd7e-4666-a9d8-cee01ae96f4d.png">

<img width="831" alt="image" src="https://user-images.githubusercontent.com/22700839/221169934-718fcfd4-58cc-4341-b81c-237b6d359361.png">

<img width="532" alt="image" src="https://user-images.githubusercontent.com/22700839/221170224-c5e3f78d-c5b8-4f40-8eb5-59bdeeffd034.png">

<img width="334" alt="image" src="https://user-images.githubusercontent.com/22700839/221170390-226b551c-8c0f-4242-b449-d5f4f38e6155.png">

### Describe the results you received and expected

I would like to know how to free up space in this directory
/var/lib/containerd/io.containerd.content.v1.content/blobs/sha256

### What version of nerdctl are you using?

nerdctl version 0.23.0

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

**Comment by fahedouch** [comment]
`nerdctl images` display images on the `default` namespace. you may have other(s) namespace that occupying this disk space ( e.g `k8s.io` => `nerdctl --namespace k8s.io images`), `k8s.io` is for kubernetes oci images

**Comment by djdongjin** [comment]
FYI: `nerdctl ns ls` will show images in every namespace

```shell
$ sudo nerdctl ns ls
NAME            CONTAINERS    IMAGES    VOLUMES    LABELS
buildkit        0             0         0
default         7             4         12
moby            1             0         0
nerdctl-test    0             3         6
```
